rst/architecture/index.rst

```diff
@@ -0,0 +1,10 @@
+#####################
+Architecture Design
+#####################
+
+HolmesProcessing architechure is based on Skald ( A Scalable Architecture for Feature Extraction, Multi-User Analysis, and Real-Time Information Sharing)
```

Comment:
typos: 
- Architecture
- SKALD
- no space after the (

might also be good to link to the paper. research gate, a uni repository, or upload the paper to the repo if you need a link